align/schema/pdk.py

```diff
@@ -0,0 +1,49 @@
+from typing import List, Union, Dict
+from align.schema import schema
+
+
+class ParasiticValues(schema.BaseModel):
+    mean: int = 0
+    min: int = 0
+    max: int = 0
+
+
+class Layer(schema.BaseModel):
+    name: str
+    gds_layer_number: int
+    gds_data_type: dict = {"draw": 0}
```

Comment:
We may be able to get a bit more specific here
```python
# Try replacing
gds_data_type: dict = {"draw": 0}
# With
gds_data_type: Dict[Literal['draw'], int] = {"draw": 0}
# Also it is dangerous to create a dict as a class attribute. Instead:
gds_data_type: Dict[Literal['draw'], int] = Field(default_factory=lambda: {"draw": 0})
```

Field must be imported from pydantic and Literal from typing (python 3.8+) or typing_extensions (python3.6)

Reply:
I am enjoying pydantic, thank you! :) I will start using Literal when we move to the new Python version in master.